repository/kafka/operator/operator.yaml

```diff
@@ -22,4 +32,15 @@ plans:
         steps:
           - name: deploy
             tasks:
-              - deploy
\ No newline at end of file
+              - deploy
+  # this plan is triggered when some parameter present in limitations is updated
+  # https://github.com/kudobuilder/operators/blob/master/repository/kafka/docs/latest/limitations.md
+  not-allowed:
+    strategy: serial
+    phases:
+    - name: not-allowed
```

Comment:
shouldn't KUDO have something built in for this? It would be better if we could mark params as `cannot-be-updated` or something and KUDO would take care of either running this plan or returning an error?

Reply:
rather than having each operator having to implement their own `not-allowed` kind of plan

Reply:
https://kubernetes.slack.com/archives/CG3HTFCMV/p1564063181112400

yeah it will be removed once KUDO provides us some solution. Like a fix for https://github.com/kudobuilder/kudo/issues/652

Reply:
But as for now I don't want users to run into a statefulset with errors.